src/index.js

```diff
@@ -12,7 +12,7 @@ function Dotdotdot() {
 }
 
 Dotdotdot.prototype = Object.create(React.Component.prototype);
-Dotdotdot.prototype.componentDidMount = function() {
+Dotdotdot.prototype.componentDidUpdate = function() {
```

Comment:
Thanks :)
With react-pure-render just componentDidMount is called, without further componentDidUpdate.